src/net/unixsock_readmsg_other.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// go:build js,wasm windows
```

Comment:
It's hard to find any references around this (it's not even mentioned in the in-progress 1.17 release notes), but assuming this works like the other directives, there should not be a space between `//` and `go`. As currently written I think this directive is being entirely ignored by the compiler.